Readme.rst

```diff
@@ -22,8 +22,8 @@
 .. |Forum| image:: https://img.shields.io/badge/forum-Discourse-39c6ff.svg
    :target: https://discuss.neos.io/
    :alt: Discussion Forum
-.. |Issues| image:: https://img.shields.io/badge/issues-Jira-39c6ff.svg
-   :target: https://jira.neos.io/browse/NEOS
+.. |Issues| image:: https://img.shields.io/github/issues/neos/neos-development-collection.svg
+   :target: https://github.com/neos/neos-development-collection/issues
```

Comment:
one could argue it's unnecessary to link to Github issues from Github itself, but since the Readme is pulled to Packagist as well it makes sense